<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseLocalFile</class>
 <widget class="QWidget" name="ChooseLocalFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChooseLocalFile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QLabel" name="showLocalFile">
       <property name="text">
        <string>发送文件路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirmLocalFile">
       <property name="text">
        <string>确认选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
